File: dotnet/src/Lang/Otterop/Lang/Panic.cs
namespace Otterop.Lang
{
    public class Panic
    {
        public static void IndexOutOfBounds(String message)
        {
            throw new IndexOutOfRangeException(message.Unwrap());
        }

        public static void InvalidOperation(String message)
        {
            throw new InvalidOperationException(message.Unwrap());
        }

    }

}

File: ts/lang/otterop/lang/Panic.ts
import { String } from './String';

export class Panic {

    public static indexOutOfBounds(message : String) : void {
        throw new Error("indexOutOfBounds");
    }

    public static invalidOperation(message : String) : void {
        throw new Error("invalidOperation: " + message);
    }

}

